src/common/languageinfo.cpp

```diff
@@ -1177,6 +1223,665 @@ tabScriptData[] =
     { nullptr, nullptr }
 };
 
+// Data table for likely subtags
```

Comment:
I think it would be nicer to move this data out to a separate file and `#include` it from here.